dbms/src/Flash/Coprocessor/DAGContext.h

```diff
@@ -412,6 +432,9 @@ class DAGContext
     // - Stream: execute with block input stream
     // - Pipeline: execute with pipeline model
     ExecutionMode execution_mode = ExecutionMode::None;
+
+    std::shared_ptr<MPPTaskOperatorSpillContexts> operator_spill_contexts;
+    std::shared_ptr<MPPQueryOperatorSpillContexts> query_operator_spill_contexts;
```

Comment:
Can "mpp" be removed?
Because DAGContext is also used by cop/batchCop.
For cop/batchCop, only one TaskOperatorSpillContexts can be registered under one query_operator_spill_contexts

Reply:
I think it can be removed, but auto spill for cop/batch cop is not considered in this pr :(

Reply:
ok